### DISCUSSION TITLE
Sunsetting cask-versions/calibre4?

### DISCUSSION THREAD

**Discussion by ChrisBaker97** [Homebrew/discussions#4674] [Casks] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

I originally [contributed](https://github.com/Homebrew/homebrew-cask-versions/pull/9850) `calibre4` to `homebrew/cask-versions` due to a number of popular plug-ins not initially being available for Calibre 5, which required Python 3. Calibre is now on v6 and all the plug-ins I use have been migrated to Python 3 for some time. I am not sure how many Homebrew users are still using Calibre 4, but I think it might be time to talk about removing it from `cask-versions`.

### Relevant casks

cask-versions/calibre4

**Comment by SMillerDev** [comment, marked as answer]
Please make a pull request for this

**Comment by ChrisBaker97** [comment]
PR [#17248](https://github.com/Homebrew/homebrew-cask-versions/pull/17248) created.
